src/app/query/bitcoin/ordinals/use-inscription-by-txid.query.ts

```diff
@@ -22,10 +20,14 @@ function makeInscriptionMetadataQueryKey(txid: string) {
   return [QueryPrefixes.InscriptionFromTxid, txid] as const;
 }
 
-export function useInscriptionByTxidQuery(txid: string) {
-  return useQuery({
+export function createQuery(txid: string) {
```

Comment:
Let's use the name that you've renamed in the import here. This is a v non-descriptive name.

Reply:
Was thinking to better use `createQueryOptions` given that's what is actually being created. The import alias is due to a naming collision with another variable with the same name from a different file. Wouldn't using it here mean that this variable name is aware of another variable from the other file? Wdyt?